apps/src/common.rs

```diff
@@ -1114,18 +1125,35 @@ impl Http3Conn {
 
 impl HttpConn for Http3Conn {
     fn send_requests(
-        &mut self, conn: &mut quiche::Connection, target_path: &Option<String>,
+        &mut self, conn: &mut quiche::Connection, client_args: &ClientArgs,
     ) {
+        let ClientArgs {
+            dump_response_path: target_path,
+            reqs_concurrent,
+            ..
+        } = client_args;
```

Comment:
Same comment here about destructuring, it causes some of the later code to use refs when it doesn't (and vice versa). Easier to just drop it for clarity.